data/build.gradle

```diff
@@ -9,11 +9,11 @@ jar {
 dependencies {
   implementation project(':util')
   implementation project(':ethereum:datastructures')
+  implementation 'com.fasterxml.jackson.core:jackson-databind'
```

Comment:
curious why you bothered moving this.  its better to keep 3rd party dependencies grouped.  actually i dont give af, but people judge

Reply:
Totally by accident.